protocol/communities_messenger_shared_member_address_test.go

```diff
@@ -819,6 +820,8 @@ func (s *MessengerCommunitiesSharedMemberAddressSuite) TestTokenMasterReceivesAc
 }
 
 func (s *MessengerCommunitiesSharedMemberAddressSuite) TestMemberReceivesPendingRequestToJoinAfterAfterGettingTokenMasterRole() {
+	s.T().Skip("flaky test")
```

Comment:
Should we not just remove flaky tests!?

Reply:
@J-Son89 no, I cannot do that here, cause that commit is one of the commits merged to `develop` before the commit I wanted to cherry-pick here, so I had to include that one as well.

I haven't modified any of the commits you see in the commits tab.